video-templates/styles/cf-video-overlay-styles.css

```diff
@@ -0,0 +1,50 @@
+/** Responsive **/
+.embed-responsive {
```

Comment:
@royboy789 Please prefix all CSS classes with .cf_video_overlays or use .cf_video_overlays .class-name These classes are likely to be used in other plugins, themes, etc.